packages/delir/domain/Renderer/PluginScriptLoader.ts

```diff
@@ -0,0 +1,46 @@
+import * as Delir from '@ragg/delir-core'
+import * as fs from 'fs'
+import Module = require('module')
+import * as path from 'path'
+import * as vm from 'vm'
+
+export default class PluginScriptLoader {
+    public static load(path: string) {
```

Comment:
The name of argument `path` should be avoided here because it is confusing with imported `path` module name.

Reply:
Tashikani